polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -270,6 +270,10 @@ pub mod pallet {
 		/// implementation should be the same as `Balance` as used in the `Configuration`.
 		type Currency: ReservableCurrency<Self::AccountId>;
 
+		/// The default channel size and capacity to use when opening a channel to a system
+		/// parachain.
+		type DefaultChannelSizeAndCapacityToSystem: Get<(u32, u32)>;
```

Comment:
I would add an `integrity_test` that these are below the maximum allowed by `Configuration`.

Reply:
config is from storages so it is not really possible to assert it at build time.
if the value is above the max defined in config, the call will fail, and I think that's fine.

Reply:
Ah right 👍

Reply:
We do run `integrity_test` in externalities, so at least the genesis values can be checked.

Reply:
but why check with genesis values?

Reply:
Because they should also be correct? but its not a must, just an idea.